sops/__init__.py

```diff
@@ -239,7 +231,7 @@ def detect_filetype(file):
     return 'text'
 
 
-def initialize_tree(path, itype):
+def initialize_tree(path, itype, kms_arns, pgp_fps):
```

Comment:
any chance I could get this to be `def initialize_tree(path, itype, kms_arns="", pgp_fps=""):` so that api calls from other modules don't have to specify those for decryption?